app/routes/github.js

```diff
@@ -0,0 +1,42 @@
+import Ember from 'ember';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+
+const {
+  get,
+  inject: { service },
+  Route
+} = Ember;
+
+const CODE_INVALID = 'The GitHub authorization code is invalid.';
+
+export default Route.extend(AuthenticatedRouteMixin, {
+  queryParams: { code: { refreshModel: true } },
```

Comment:
yeah if I am understanding this right, down in the model hook we are extracting `code` from the `params` argument and not from the `queryParams`. is that we need the `refreshModel` for something?